cmd/lncli/main.go

```diff
@@ -380,6 +386,11 @@ func main() {
 				"to lnd. This flag may be specified multiple " +
 				"times. The format is: \"key:value\".",
 		},
+		cli.BoolFlag{
+			Name: "insecure",
+			Usage: "Connect to the rpc server without TLS " +
```

Comment:
Should we hide this flag by default (`Hidden: true`)? Since it won't ever work with an `lnd` directly and people might get confused by it (there's no way to disable TLS on the RPC interface, only REST or through `litd`).